core-schema/src/main/resources/sql-schema/V6__Mod_kitpvp.sql

```diff
@@ -57,10 +54,24 @@ CREATE TABLE kitpvp_kits_contents (
   CONSTRAINT nonnegative_slot CHECK (slot >= 0)
 );
 
+CREATE TABLE kitpvp_bounties (
+  user_id INT NOT NULL,
+  bounty_amount NUMERIC(15, 3) NOT NULL DEFAULT 0,
+  bounty_currency TINYINT NOT NULL DEFAULT 0,
+
+  FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
+  INDEX bounty_amount_index (bounty_amount),
```

Comment:
Index currency instead of amount?

Reply:
There is an index on the amount in order to select the top bounties for the bounty list.

The currency doesn't deserve an index because there are only 2 possible values. An index on the currency column would not visibly speed up SELECT queries but it would consume more memory and slow down INSERT queries.